sched/task/task_setup.c

```diff
@@ -155,6 +155,7 @@ static int nxtask_assign_pid(FAR struct tcb_s *tcb)
 
   for (i = 0; i < g_npidhash / 2; i++)
     {
+      DEBUGASSERT(g_pidhash[i] != NULL);
```

Comment:
no need to check NULL here. line 158 will trigger an NULL pointer exception immediately if` g_pidhash[i]` equal NULL